<launch>
   <node name="whycon_left" type="whycon" pkg="whycon">
    <remap from="image_raw" to="/trigger/left/image_color"/>
    <param name="targets" value="1"/>
    <param name="width" value="640"/>
    <param name="height" value="480"/>
    <param name="camera_name" value="00b09d0100cd6ea4"/>
  </node>
  <node name="whycon_right" type="whycon" pkg="whycon">
    <remap from="image_raw" to="/trigger/right/image_color"/>
    <param name="targets" value="1"/>
    <param name="width" value="640"/>
    <param name="height" value="480"/>
    <param name="camera_name" value="00b09d0100cd6e9d"/>
  </node>
 
  <node name="triangulator" type="triangulator" pkg="whycon" output="screen">
    <param name="camera_left_name" value="00b09d0100cd6ea4"/>
    <param name="camera_right_name" value="00b09d0100cd6e9d"/>
  </node>
</launch>
  
